src/app/modules/core/manager/landlord/components/search/search.component.html

```diff
@@ -4,7 +4,8 @@
       <span>Search</span>
     </mat-toolbar>
   </div>
-  <div class="inputs" fxLayout="row" fxLayoutGap="20px" fxLayoutAlign="center center">
+  <div class="inputs" fxLayout="row" fxLayout.lt-md="column" fxLayoutGap="4%"
```

Comment:
Sul mobile non c'e' spazio fra le cards quando sono allineate in verticale.
![image](https://user-images.githubusercontent.com/66738419/99897212-fe1e1400-2c97-11eb-8a72-2d255d66d769.png)